N/NOMAD/build_tarballs.jl

```diff
@@ -5,24 +5,18 @@ version = v"4.0.0"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://gist.github.com/amontoison/06dac8b63424854f754264597af6b09e/raw/942aba554a82800d0c7438288d3da1a827ef2974/NOMAD.zip", "a6653af375be8006e742239af3914ba48034015d57a2be7a07fc14c6ff245d1b"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/amontoison/nomad.git","4c5511e873fc24b82f5ea9ffccd9c9912b4eebfe"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd "${WORKSPACE}/srcdir/NOMAD"
```

Comment:
I assume this is now called
```suggestion
cd "${WORKSPACE}/srcdir/nomad"
```

Reply:
Thanks !